### PR TITLE
Play 2.9 & Scala 3

### PR DESCRIPTION
**Pull request by AshCorr**
## What this changes

Play 2.9 was just released, this PR bumps `play-mockws` to the newest version. Play 3.x has also just been released but will involve some breaking changes, figure some people might have some benefit to having a Play 2.9 version of this library before then.

## How this was tested

Tests ran against Scala 2.13 and Scala 3 with:

`sbt ++3.3.1! test`
`sbt ++2.13.1! test`

### CODE REVIEW

File: RELEASE-NOTES.md
Comment (AshCorr) [inline RELEASE-NOTES.md:1]: Included the changes to `RELEASE-NOTES.md` and `version.sbt` to release 2.9, but feel free to drop these changes if you want to make them yourself!

File: project/build.properties [hunk 1]
```diff
@@ -1 +1 @@
-sbt.version = 1.3.12
+sbt.version = 1.5.0
```
Comment (AshCorr) [inline project/build.properties:1]: Scala 3 requires atleast SBT 1.5

File: src/test/scala/mockws/AuthenticationTest.scala [hunk 2]
```diff
@@ -45,16 +45,16 @@ class AuthenticationTest extends AnyFunSuite with Matchers with ScalaCheckProper
     }
 
     a[UnsupportedOperationException] shouldBe thrownBy(
-      await(ws.url("/").withAuth("user", "s3cr3t", WSAuthScheme.NTLM).get)
+      await(ws.url("/").withAuth("user", "s3cr3t", WSAuthScheme.NTLM).get())
```
Comment (AshCorr) [inline src/test/scala/mockws/AuthenticationTest.scala:48]: Functions must be called using parenthesis in Scala now

File: src/test/scala/mockws/MockWSTest.scala [hunk 3]
```diff
@@ -20,6 +20,9 @@ import scala.concurrent.Future
 import scala.concurrent.duration._
 import org.scalatest.funsuite.AnyFunSuite
 import org.scalatest.matchers.should.Matchers
+import play.api.libs.ws.writeableOf_String
```
Comment (AshCorr) [inline src/test/scala/mockws/MockWSTest.scala:23]: These are implicit BodyParser implementations for reading and writting Strings to request bodies. They now need to be imported explicitly in Scala 3 (I think? Scala 3 compilation wouldn't work unless I did).

File: project/build.properties (hunk 1 above)
Comment (gaeljw) [reply project/build.properties:1]: Play 2.9 requires at least sbt 1.9.6, not sure how this affect libraries such as this one though but upgrading to sbt 1.9.7 should be straightforward, isn't it?

File: src/test/scala/mockws/MockWSTest.scala (hunk 3 above)
Comment (gaeljw) [reply src/test/scala/mockws/MockWSTest.scala:23]: I had the same "issue" in my projects, not sure if that's really intended though.

File: project/build.properties (hunk 1 above)
Comment (AshCorr) [reply project/build.properties]: Ah good catch! Seems to compile fine and tests run, so lets bump it.

https://www.playframework.com/documentation/3.0.x/Highlights29#Scala-2.12,-sbt-0.13-and-Java-8-Support-discontinued

**Review by gaeljw — APPROVED** [review]

**Review by avdv — APPROVED** [review]
Thank you, LGTM.

File: .github/workflows/ci.yml [hunk 4]
```diff
@@ -22,21 +22,21 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest]
-        scala: [2.12.10, 2.13.1]
-        java: [temurin@8]
+        scala: [2.13.1, 3.3.1]
```
Comment (mkurz) [inline .github/workflows/ci.yml:25]: Why 2.13.1? The latest 2.13.x release is 2.13.12
Is this a typo, like everywhere in this PR?

Comment (AshCorr) [reply .github/workflows/ci.yml:25]: [2.13.1 was the previously set version of Scala](https://github.com/leanovate/play-mockws/blob/88a0c36e98bab7c810b292c5f5980c221e0dbc9d/build.sbt#L9) that this project was using, I avoided bumping anything more than I needed to get this project to work with Play 2.9 and Scala 3

**Pull request merged**